zerver/actions/realm_settings.py

```diff
@@ -389,7 +390,8 @@ def do_delete_all_realm_attachments(realm: Realm, *, batch_size: int = 1000) ->
 
 def do_scrub_realm(realm: Realm, *, acting_user: Optional[UserProfile]) -> None:
     if settings.BILLING_ENABLED:
-        downgrade_now_without_creating_additional_invoices(realm)
+        billing_session = RealmBillingSession(user=None, realm=realm)
```

Comment:
I think you want `user=acting_user` in this file as well. The point of the `user` parameter here is to record for our audit logs, which record (both for billing and otherwise) which individual made a given change, and we should set it whenever we know it. We should always set it to that user.

Might be worth renaming the `RealmBillingSession` parameter to `acting_user` to make this more obvious.

Reply:
> Might be worth renaming the RealmBillingSession parameter to acting_user to make this more obvious.

I'll do this as a follow-up, as I need to address one more follow-up from the `make_end_of_cycle_updates_if_needed` PR, that would b related to this.

Reply:
OK